packages/components/src/internal/components/search/SampleFinderSection.tsx

```diff
@@ -0,0 +1,183 @@
+        )}
+        >
+            {filterCards.length == 0 ?
+                <>
+                    <FilterCards className="empty" cards={parentEntityDataTypes.map((entityDataType) => ({
+                        entityDataType,
+                        onAdd: onAddEntity
+                    }))} />
+                    <div className="filter-hint">{getFinderStartText(parentEntityDataTypes)}</div>
+                </>
+                :
+                <>
+                    <FilterCards cards={filterCards} />
+                    <SampleFinderSamples
+                        {...gridProps}
+                        cards={filterCards}
+                    />
+                </>
+            }
+            {showFilterModal && (
+                <EntityFieldFilterModal
+                    onCancel={onFilterClose}
+                    entityDataType={chosenEntityType}
+                    onFind={onFind}
+                />
+            )}
+        </Section>
+    )
+});
+
+interface SampleFinderSamplesProps extends SampleFinderSamplesGridProps {
+    cards: FilterCardProps[];
+}
+
+export const SampleFinderSamplesImpl: FC<SampleFinderSamplesProps & InjectedQueryModels> = memo(props => {
+    const { actions, queryModels } = props;
+
+    return(
+        <>
+            <h4>Filtering and sample-type tabs coming soon...</h4>
+            <SamplesTabbedGridPanel
+                {...props}
+                withTitle={false}
+                asPanel={false}
+                actions={actions}
+                queryModels={queryModels}
+                tabbedGridPanelProps={{
+                    alwaysShowTabs: true,
+                    advancedExportOptions: SAMPLE_DATA_EXPORT_CONFIG,
+                }}
+            />
+        </>
+    )
+});
+
+// exported for jest testing
+export const SampleFinderSamplesWithQueryModels = withQueryModels<SampleFinderSamplesProps>(SampleFinderSamplesImpl);
+
+
+// exported for jest testing
+export const SampleFinderSamples: FC<SampleFinderSamplesProps> = memo((props) => {
+    const { cards } = props;
+
+    const baseFilters = [];
+    // TODO this is not really correct.  Probably there will be a specialized filter type used for these lineage queries.
+    // cards.forEach(card => {
+    //     if (card.filterArray.length) {
+    //         baseFilters.push(...card.filterArray);
+    //     } else  {
+    //         baseFilters.push(card.entityDataType.inputColumnName.replace("First", card.schemaQuery.queryName), null, Filter.Types.NONBLANK);
+    //     }
+    // });
+
+    const queryConfigs = {
+        'allSamples': {
+            id: 'allSamples',
+            title: 'All Samples',
+            schemaQuery: SCHEMAS.EXP_TABLES.MATERIALS,
+            requiredColumns: SAMPLE_STATUS_REQUIRED_COLUMNS,
+        }
+    }
+
+    return (
+        <SampleFinderSamplesWithQueryModels
+            {...props}
```

Comment:
minor: does the downstream components actually use the `cards` prop? If not, could do `const { cards, ...gridProps } = props;` and then pass `gridProps` here.

Reply:
Done.